test_rclcpp/test/test_allocator.cpp

```diff
@@ -34,9 +34,30 @@
 
 #endif  // not WIN32
 
+static const size_t num_rmw_tokens = 5;
+static const char * rmw_tokens[num_rmw_tokens] = {"librmw", "dds", "DDS", "dcps", "DCPS"};
```

Comment:
Do you have any suggestions for how I can implement ignoring function calls from within the rmw implementation?